### ISSUE TITLE
questions about log_prob implementation?

### ISSUE THREAD

**Issue by dragen1860** [wongongv/PolicyGradient_in_tensorflow2.0#1]
Hi, thanks for your PG implementation.
 I found it's difficult to understand this episode code:
```python
	#train model
	def log_prob(self, policy_param, acs):
		if self.is_discrete:
			logits = policy_param
			log_prob = tf.keras.losses.sparse_categorical_crossentropy(\
				y_true = acs, y_pred = logits, from_logits = True)
```
I think the log_prob function will just return `tf.math.log(policy_param)` and I do not understand why you calculate crossentropy loss here? Thank you.
